kits/beta-base/contracts/BetaKitBase.sol

```diff
@@ -0,0 +1,160 @@
+        bytes32[4] _appIds
+    )
+        KitBase(_fac, _ens)
+        public
+    {
+        require(isContract(address(_fac.regFactory())));
+
+        minimeFac = _minimeFac;
+        aragonID = _aragonID;
+        appIds = _appIds;
+    }
+
+    function createDAO(
+        string name,
+        MiniMeToken token,
+        address[] holders,
+        uint256[] stakes,
+        uint256 _maxTokens
+    )
+        internal
+        returns (Kernel)
+    {
+        require(holders.length == stakes.length);
+
+        Kernel dao = fac.newDAO(this);
+
+        ACL acl = ACL(dao.acl());
+
+        acl.createPermission(this, dao, dao.APP_MANAGER_ROLE(), this);
+
+        Voting voting = Voting(
+            dao.newAppInstance(
+                appIds[uint8(Apps.Voting)],
+                latestVersionAppBase(appIds[uint8(Apps.Voting)]),
+                new bytes(0),
+                true
+            )
+        );
+        emit InstalledApp(voting, appIds[uint8(Apps.Voting)]);
+
+        Vault vault = Vault(
+            dao.newAppInstance(
+                appIds[uint8(Apps.Vault)],
+                latestVersionAppBase(appIds[uint8(Apps.Vault)]),
+                new bytes(0),
+                true
+            )
+        );
+        emit InstalledApp(vault, appIds[uint8(Apps.Vault)]);
+
+        Finance finance = Finance(
+            dao.newAppInstance(
+                appIds[uint8(Apps.Finance)],
+                latestVersionAppBase(appIds[uint8(Apps.Finance)]),
+                new bytes(0),
+                true
+            )
+        );
+        emit InstalledApp(finance, appIds[uint8(Apps.Finance)]);
+
+        TokenManager tokenManager = TokenManager(
+            dao.newAppInstance(
+                appIds[uint8(Apps.TokenManager)],
+                latestVersionAppBase(appIds[uint8(Apps.TokenManager)]),
+                new bytes(0),
+                true
+            )
+        );
+        emit InstalledApp(tokenManager, appIds[uint8(Apps.TokenManager)]);
+
+        // Required for initializing the Token Manager
+        token.changeController(tokenManager);
+
+        // Permissions
+        acl.createPermission(acl.ANY_ENTITY(), voting, voting.CREATE_VOTES_ROLE(), voting);
+        acl.createPermission(voting, voting, voting.MODIFY_QUORUM_ROLE(), voting);
+
+        acl.createPermission(finance, vault, vault.TRANSFER_ROLE(), voting);
+        acl.createPermission(voting, finance, finance.CREATE_PAYMENTS_ROLE(), voting);
+        acl.createPermission(voting, finance, finance.EXECUTE_PAYMENTS_ROLE(), voting);
+        acl.createPermission(voting, finance, finance.DISABLE_PAYMENTS_ROLE(), voting);
+        acl.createPermission(voting, tokenManager, tokenManager.ASSIGN_ROLE(), voting);
+        acl.createPermission(voting, tokenManager, tokenManager.REVOKE_VESTINGS_ROLE(), voting);
+
+        // App inits
+        vault.initialize();
+        finance.initialize(vault, uint64(-1) - uint64(now)); // yuge period
```

Comment:
I think we should set a reasonable period duration like 1 month, otherwise no DAO will be able to start using accounting periods in the future as the first period will never end and [changing period duration only affects the next period](https://github.com/aragon/aragon-apps/blob/master/apps/finance/contracts/Finance.sol#L550).